### DISCUSSION TITLE
Can't import cf_units, "symbol not found" error.

### DISCUSSION THREAD

**Discussion by Rocio-CP** [SciTools/cf-units#296] [Q&A] [answered]
Hi all,

Not-really-technical person here. When I try to import cf_units I get this error:
`dlopen(/Users/rocio/.local/share/virtualenvs/CMEMS-REP-c2e6tGM8/lib/python3.10/site-packages/cf_units/_udunits2.cpython-310-darwin.so, 0x0002): symbol not found in flat namespace (_ut_ignore)`
cf_units v3.1.1 installed in a pipenv environment via pipenv install cf_units in a macOS Monterey 12.6, Apple M1; Python 3.10.6

I have found crumbs of information here and there while searching, but nothing cf_units specific, and some of the solutions' wording for other python packages went _whoosh_ over my head. 

Thanks for the help!
Cheers,
Rocío

**Comment by trexfeathers** [comment, marked as answer]
Thanks for getting in touch @Rocio-CP, looking at the [PyPI files](https://pypi.org/project/cf-units/#files) - we don't publish a build for macOS `v12`. This is the default behaviour for our builder ([cibuildwheel](https://github.com/pypa/cibuildwheel)), so I assume support for `v12` will come in time. @bjlittle should we explore explicitly supporting `v12` in the workflow config?

**Reply by Rocio-CP** [reply]
Thank you! I'll keep an eye, then.
